### PR TITLE
:sparkles: Content development page module#89

### LINKED ISSUE #89: ✨📝 Create The Content Development Page Module
## User stories

As a Developer I need a Page Module where I can create declarations(components, directives...) that will represent the different sections of my content-development page.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Description

As a Developer I need a Page Module where I can create declarations(components, directives...) that will represent the different sections of my content-development page.

### Technical Analysis

The module should be placed in the library `libs/features/pages/content-development` (content-development is a module). The module is named "ContentDevelopmentModule". The module has one component called `content-development-page` found in the `lib/main` folder. For reference check how this has been implemented in the other page modules. 

The route to this should be set in the root routes and it should also have a child routing module that renders the `content-development-page`  component

### Acceptance Criteria

- [ ] The code is isolated to the `libs/features/pages/content-development` package unless a justified reason is specified.
- [ ] A separate commit is used for each small change
- [ ] The commits and PR are well formatted/use proper documentation

### PR DESCRIPTION
**Pull request by Marionwanyama23**
# Description

This task contains a Page Module where declarations(components, directives...) can be created that will represent the different sections of the content-development page.

Fixes #89 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/pages/content-development/src/lib/lib.routes.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
```
Comment (LemmyMwaura) [inline libs/features/pages/content-development/src/lib/lib.routes.ts:1]: rename this file to `content-development.routing.ts`

File: libs/features/pages/content-development/src/lib/lib.routes.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { RouterModule, Route } from '@angular/router';
+
+import { ContentDevelopmentPageComponent } from './main/content-development-page/content-development-page.component';
+export const ELEWA_CONTENT_DEVELOPMENT_ROUTES: Route[] = [
```
Comment (LemmyMwaura) [inline libs/features/pages/content-development/src/lib/lib.routes.ts:5]: add a space between these two

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Just two small changes left

File: libs/features/pages/content-development/src/lib/lib.routes.ts (hunk 2 above)
Comment (Marionwanyama23) [reply libs/features/pages/content-development/src/lib/lib.routes.ts]: Changes requested have been made

File: libs/features/pages/content-development/src/lib/lib.routes.ts (hunk 1 above)
Comment (Marionwanyama23) [reply libs/features/pages/content-development/src/lib/lib.routes.ts]: Changes requested have been made

**Review by LemmyMwaura — APPROVED** [review]

**Pull request merged**